### PR TITLE
Singhriddhi075.md

### PR DESCRIPTION
**Pull request by Singhriddhi075**

### CODE REVIEW

**Comment by atulsuresh** [comment]
@Singhriddhi075 can you change the file name from first-opensrccontribution to yourUserName.md?

it would be Singhriddhi075.md

**Comment by Singhriddhi075** [comment]
ohk.I'll change this.

**Comment by Singhriddhi075** [comment]
done!

**Comment by atulsuresh** [comment]
you should change the name of the file not the pull request. 

Rename the file you created to Singhriddhi075.md and commit and the new file. 

You can check into https://docs.github.com/en/github/managing-files-in-a-repository/renaming-a-file

**Comment by Singhriddhi075** [comment]
is it done?

File: src/Singhriddhi075.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+username:"Singhriddhi075"
+name:Riddhi singh"
```
Comment (stanly-johnson) [inline src/Singhriddhi075.md:3]:
```suggestion
name:"Riddhi singh"
```

**Review by stanly-johnson — APPROVED** [review]

**Comment by Singhriddhi075** [comment]
Now what I have to do after this?

**Comment by stanly-johnson** [comment]
@Singhriddhi075 Now that you have completed the basic course. You can choose any other project within Leapcode and work on an issue.

**Pull request merged**